payment-manager/src/main/resources/db/migration/V1.0__CREATE_WALLETS_TABLE.sql

```diff
@@ -7,4 +7,6 @@ CREATE TABLE IF NOT EXISTS wallets
     user_type  VARCHAR(32),
     created_at TIMESTAMP(6),
     updated_at TIMESTAMP(6)
-);
\ No newline at end of file
+);
+
+CREATE INDEX IF NOT EXISTS user_id_index ON wallets(user_id);
```

Comment:
Actually this script has been merged. it's kind of in "production". So, you cannot rewrite this script, you can only create a new one. But this time, ok, you can leave as it is

Reply:
Do you mean the whole feature in general ?

Reply:
getWalletByUser method

Reply:
As you've already done it, merge it